prd.md

```diff
@@ -0,0 +1,14 @@
+# final-project prd
+
+Fishtown development Map and Timeline
+
+Purpose: The dashboard I will be creating will be helpful for Fishtown residents to make sense of the real estate development happening in their neighborhood. Since Philadelphia City Council instated a tax-abatment around the transit cooridor of the Market Frankford Line, Fishtown has seen a substansial amount of investment and construction on its properties. This increase in development is exciting to some, but threatening to others. My dashboard serves to help all groups understand the situation. Furthermore, it could be utilized by the Fishtown Neighbors Association Zoning Board in their meetings, to help everyone know exactly where a specific development is. Finally, if a resident is walking down the street and see a construction site, they can consult this dashboard to learn more details about it. The pace of development in Fishtown can be disorienting to some, and this dashboard aims to bring clarity to the situation. 
+
+Features: This dashboard will be developed primarily for desktop usage. It's main feature is a map of building outlines and other landmarks within the Fishtown Neighbors Association's border. Building outlines will be color coded by if they are under construction, was constructed since the tax abatement was instated, and if they were constructed before the abatement. Under construction buildings can be clicked on and information about the construction will be displayed, including building details, development group, etc. A timeline scroll bar can be adjusted to see the developments take place over time. 
+
+Milestones: Collecting and tidying data, map construction with color fills of building outlines. Information for development displayed on click. I am open to extra features from the FNA Zoning Board.
+
+Data:
+Building footprints: https://www.opendataphilly.org/dataset/buildings
+Development data from L&I: https://www.opendataphilly.org/dataset/licenses-and-inspections-building-certifications
```

Comment:
My hunch is that the building & zoning permits dataset from L&I may be more helpful for this purpose: https://www.opendataphilly.org/dataset/licenses-and-inspections-building-permits. And maybe demolitions: https://www.opendataphilly.org/dataset/building-demolitions. Either way, there will probably be a fair bit of data preparation needed. But it sounds like a fun challenge! Happy to chat through it as needed.

Reply:
Cool. I'll be comfortable doing the data prep in R, incorporating those other L&I datasets too. Putting it into JS will be more of a challenge!